### ISSUE TITLE
imap_check_subscribed broken by 978cf231a9d45e530f157e93e79f3549dc0fbf7f

### ISSUE THREAD

**Issue by nabijaczleweli** [neomutt/neomutt#2620]
## Expected Behaviour

I can see my mailboxes.

## Actual Behaviour

They don't show up.

## Steps to Reproduce

Open a +INBOX on an IMAP server with subscriptions.

## How often does this happen?

- Always

## When did it start to happen?

- When I upgraded
  Which version did you use to use?

`20200821` -> 978cf231a9d45e530f157e93e79f3549dc0fbf7f 

## Extra Info

```
$ git bisect start HEAD 20200821
# ...
$ git bisect good
978cf231a9d45e530f157e93e79f3549dc0fbf7f is the first bad commit
commit 978cf231a9d45e530f157e93e79f3549dc0fbf7f
Author: Richard Russon <rich@flatcap.org>
Date:   Wed Aug 12 17:48:00 2020 +0100

    sidebar: strip sidebar update functions

    Remove the hard-coded sidebar update functions.
    The sidebar now relies on notifications / observer functions.

:100644 100644 ce8f44c2c716d74491dd63966eda2c20e7570436 b9f4f37cce98360181bcb3b0bb52b0df28733e7f M      command_parse.c
:100644 100644 7e08ac2015d2e90447a0b9975820b97348199747 358b19d8ace1531021c675db8eb469b8a0551b73 M      index.c
:100644 100644 76bca7a6e8994e998ac857fb6ab72e2d41f0b536 41e2d45800b56b504f38d8fa199ddd70089ccdb6 M      main.c
:040000 040000 404e75e721e4d16a115cf2dc549133ef3c4e9f1c 65ac3dd6165ec0bfe76d1da586d74659a1a43d71 M      sidebar
```

* Operating System and its version: Buster
* Were you using multiple copies of NeoMutt at once? no
* Were you using 'screen' or 'tmux'? no
* Is your email local (maildir) or remote (IMAP)? both, but this exemplifies on remote, obviously

**Comment by gahr** [comment]
I think @flatcap is working on a fix here #2611

**Comment by nabijaczleweli** [comment]
and I think I had commented there saying that his fixes work for me

**Comment by gahr** [comment]
hand't noticed

**Comment by gahr** [comment]
any idea how to tag these issues so one doesn't accidentally start looking into them? Or should we just close them already?

**Comment by flatcap** [comment]
Between the original post and the comments, there's a **Mention**:

> nabijaczleweli (наб) mentioned this issue 44 minutes ago
> Fix sidebar on new/removed mailboxes #2611 

Also, there's a "Linked pull requests" button (like mentioning # 123)

**Comment by gahr** [comment]
I assigned the issue to you @flatcap so I know it's beeing taken care of 👍
